ci/cases/gfsv17/C384mx025_3DVarAOWCDA.yaml

```diff
@@ -8,12 +8,13 @@ arguments:
   resdetatmos: 384
   resdetocean: 0.25
   nens: 0
-  interval: 6
+  interval: 0
```

Comment:
Refresh my memory - what does setting to 0 do to this?

Reply:
Only the gdas cycle will run, not the gfs.

Reply:
why are we reverting to no-GFS run?